js/views/userPage/Store.js

```diff
@@ -599,3 +603,8 @@ export default class extends BaseVw {
     return this;
   }
 }
+
+Store.fetchListings = (cl, options = {}) =>
```

Comment:
@jjeffryes I decided to cut a bit of a corner and not make the util which would pool Listings fetches and cache them in-memory. Turns out, the listings endpoint returns http cache headers, there was just a bug preventing it from working which has since been fixed.

To summarize the repercussions:
- I bust the cache on store fetches since we've always taken the stance with those that we were prioritizing being up to date over latency. If we feel different I could put a one hour (or whatever - it's configurable on a per request basis) cache on them (as long as it's not your own listings, which I think we'd all agree should have no cache).
- moreListings are cached for an hour
- I removed the passing around of the listings collection since it was kind of ugly and getting uglier. E.g. we were passing it into the `listings grid > listing card > listing detail`... but to really close the loop, it should also have gone into more-listings, which would pass it onto listing card, so `listings grid > listing card > listing detail > more-listings > listing card > listing detail > ...`.

It's a cached call now, so it should be fine. The slight inefficiency is that with the store fetch being uncached, clicking on a store card will end up getting you another fetch (the cached one), but then any more clicks on more-listings card in that store would have access to the cached content for an hour.

Just a case of simpler code in exchange for some mild request inefficiency.